.travis.yml

```diff
@@ -24,7 +28,10 @@ env:
   - PURE_RUBY=1
   - KITCHEN_SINK=1
 
-script: rake build && rake test:spec
+script: >
+  (bundle exec rake build) &&
+  (find . -not -path './vendor/*' -type f -iname '*.rb' -exec sed -i -z '1s/^\(\s*#\s*encoding[^\n]*\n\)\?/\1# frozen_string_literal: true\n/' '{}' +;) &&
```

Comment:
I am with you on removing this hack before merging. The hack cause stack traces to shift which would cause confusion for others. But for now, it serves as a makeshift solution, so we can review the changes with better confidence.